### ISSUE TITLE
Returns folder instead of file on Mac OS X (when a folder is selected, and "change picture" is selected)

### ISSUE THREAD

**Issue by sg-s** [sindresorhus/wallpaper#4]
running `wallpaper`
returns the path of the folder that contains the current wallpaper, instead of the path to the actual file. 

this issue is true whether or not multiple desktops are enabled.

**Comment by sindresorhus** [comment]
Can you try setting a wallpaper through System Preferences and see if it's still a folder? OS X version?

**Comment by sg-s** [comment]
hi, i did set the wallpaper using System Preferences. I'm on Mac OS X 10.10.3. node: v0.12.2

**Comment by sg-s** [comment]
ah ha! i misunderstood your question — it is set to a folder in System Preferences. But shouldn't the expected behaviour be to return the path to the current wallpaper?

**Comment by sindresorhus** [comment]
Ah, right. Was able reproduce it now when I selected "Change picture". The problem is that I'm using the [blessed OS X API](https://github.com/sindresorhus/osx-wallpaper/blob/ab3bdefe7e3f84def5b7b50ce86e4711c31ff18d/wallpaper/main.m#L29) to get the wallpaper, so not sure there's much to do about it. I agree it's suboptimal though.

I made this mainly for being able to set the wallpaper though, and just added a getter for convenience.

I'd say PR welcome [here](https://github.com/sindresorhus/osx-wallpaper), but not a priority for me.

Seems like this returns the correct wallpaper, though would prefer an Objective-C solution:

```applescript
tell application "Finder" to POSIX path of (desktop picture as alias)
```

Another lead: http://stackoverflow.com/questions/14099363/get-the-current-wallpaper-in-cocoa
